public/styles/styles.css

```diff
@@ -33,6 +39,35 @@ input {
   background-color: rgba(101,112,86, 0.2);
 }
 
+.inc, .dec {
+  height: 50px;
```

Comment:
Just a future thing to ponder -- do we want to use em/rem since we are going to be making a multi-device app?

Reply:
I was going to throw in some media queries once the full layout's finalized. There'll be a lot of little things to tweak.